---
title: MSSQLSERVER_26013 | Microsoft Docs
ms.custom: ''
ms.date: 03/06/2017
ms.prod: sql-server-2014
ms.reviewer: ''
ms.suite: ''
ms.technology:
- database-engine
ms.tgt_pltfrm: ''
ms.topic: article
helpviewer_keywords:
- 26013 (Database Engine error)
ms.assetid: 7362fca4-b005-4872-bdfa-a3e32f834db8
caps.latest.revision: 9
author: craigg-msft
ms.author: craigg
manager: jhubbard
ms.openlocfilehash: 79d3c5a9b5e9499d2b9ddd5aea92c62844615ef9
ms.sourcegitcommit: 5dd5cad0c1bbd308471d6c885f516948ad67dfcf
ms.translationtype: MT
ms.contentlocale: ja-JP
ms.lasthandoff: 06/19/2018
ms.locfileid: "36085230"
---
# <a name="mssqlserver26013"></a>MSSQLSERVER_26013
    
## <a name="details"></a>詳細  
  
|||  
|-|-|  
|製品名|SQL Server|  
|イベント ID|26013|  
|イベント ソース|MSSQLSERVER|  
|コンポーネント|SQLEngine|  
|シンボル名|SNI_SSL_READY|  
|メッセージ テキスト|暗号化用の証明書 [Cert Hash(sha1) "%hs"] が正常に読み込まれました。|  
  
## <a name="explanation"></a>説明  
 これは情報メッセージです。 名前付き証明書が [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] に正常に読み込まれました。この証明書は暗号化に使用できます。  
  
## <a name="user-action"></a>ユーザーの操作  
 必要なアクションはありません。  
  
  